example/local/project_configurations.json

```diff
@@ -23,42 +24,69 @@
         "appId" : "<your_qa_app_id>"
       }
     },
-    "retrofit" : {													                            // Optional
+    "retrofit" : {
       "timeout" : 30,
       "prod" : "<your_prod_api_url>",
       "dev" : "<your_dev_api_url>",
       "qa" : "<your_qa_api_url>"
     },
-    "others" : [													                            // Optional
+    "others" : [
       {
         "name" : "GOOGLE SUPPORT",
-        "version" : "25.3.1",
+        "version" : "27.0.1",
         "list" : [
+          "com.android.support:appcompat-v7",
           "com.android.support:recyclerview-v7",
           "com.android.support:cardview-v7",
           "com.android.support:design"
         ]
       },
       {
-        "name" : "GLIDE",
-        "version" : "3.7.0",
+        "name" : "RETROFIT",
+        "version" : "2.3.0",
         "list" : [
-          "com.github.bumptech.glide:glide"
+          "com.squareup.retrofit2:retrofit",
+          "com.squareup.retrofit2:converter-jackson"
+        ]
+      },
+      {
+        "name" : "OKHTTP",
+        "version" : "3.9.0",
+        "list" : [
+          "com.squareup.okhttp3:logging-interceptor"
+        ]
+      },
+      {
+        "name" : "LOMBOK",
+        "version" : "1.16.18",
+        "list" : [
+          "org.projectlombok:lombok"
+        ]
+      },
+      {
+        "name" : "ADAL",
+        "version" : "0.1.15",
+        "list" : [
+          "com.massivedisaster.adal:adal-accounts"
+        ]
+      },
+      {
+        "name" : "AFM",
```

Comment:
@extmkv should we reference the kotlin version here instead of the java version?